### PR TITLE
feat: add has_fileuploadfield

### PR DESCRIPTION
**Pull request by magsyg**
Added field to RecruitmentPosition of has_file_upload
Is necessary if we want to show some position pages with file upload and some without easier
Part of #982

Also added tests for this and for gang/section requirement

### CODE REVIEW

**Review by Snorre98 — COMMENTED** [review]
Se kommentar

File: backend/samfundet/models/recruitment.py [hunk 1]
```diff
@@ -185,11 +189,27 @@ def resolve_org(self, *, return_id: bool = False) -> Organization | int:
...
+            errors['section'].append(self.ONLY_ONE_OWNER_ERROR)
+        elif not (self.gang or self.section):
+            errors['gang'].append(self.NO_OWNER_ERROR)
+            errors['section'].append(self.NO_OWNER_ERROR)
+        if self.has_file_upload:
+            if not self.file_description_nb or len(self.file_description_nb) == 0:
+                errors['file_description_nb'].append(self.FILE_DESCRIPTION_REQUIRED_ERROR)
+            if not self.file_description_en or len(self.file_description_en) == 0:
+                errors['file_description_en'].append(self.FILE_DESCRIPTION_REQUIRED_ERROR)
+        raise ValidationError(errors)
```
[the remark this answers is not in the document]
Comment (Snorre98) [reply backend/samfundet/models/recruitment.py:212]: Legger bare til kommentarer

[the remark this answers is not in the document]
Comment (magsyg) [reply backend/samfundet/models/recruitment.py]: Done

[the remark this answers is not in the document]
Comment (magsyg) [reply backend/samfundet/models/recruitment.py]: Blir vell litt samme, trengte å fikse det i begge

**Review by Snorre98 — APPROVED** [review]
lgtm

**Pull request merged**